__tests__/server/middleware/sendHtml.spec.js

```diff
@@ -97,7 +97,7 @@ jest.mock('../../../src/universal/utils/transit', () => ({
 }));
 
 jest.spyOn(console, 'info').mockImplementation(() => {});
-jest.spyOn(console, 'log').mockImplementation(() => {});
+// jest.spyOn(console, 'log').mockImplementation(() => {});
```

Comment:
should this be commented out ? uncomment or remove
```suggestion
```

Reply:
@JAdshead good catch! I commented out for testing